packages/notify-slack/README.md

```diff
@@ -0,0 +1,31 @@
+# `notify-slack` - **Github Action**
+
+This action will send a slack message
+
+## Input
+
+| Name          | Description                                             |
+| ------------- |---------------------------------------------------------|
+| `CHANNEL_ID` | The slack channel id                                     |
+| `MESSAGE`       | The message display in first block of notification    |
+| `SLACK_BOT_TOKEN`       | OAuth token of your slack app                 |
+
+## Example Workflow File
+
+```yaml
+name: Notify slack
+
+on: [pull_request]
+
+jobs:
+    notify-slack:
+        runs-on: ubuntu-latest
+        steps:
+          - name: Send message to Slack channel
+            if: ${{ failure() }}
```

Comment:
Can it be parametrized in the caller ?

Reply:
Yes it's, you can put what you want in the caller (i just put this example here to show a "real" use case")